apps/api/src/services/data-provider/data-enhancer/trackinsight/trackinsight.service.ts

```diff
@@ -180,4 +168,33 @@ export class TrackinsightDataEnhancerService implements DataEnhancerInterface {
   public getTestSymbol() {
     return 'QQQ';
   }
+
+  private async searchTrackinsightSymbol({
+    requestTimeout,
+    symbol
+  }: {
+    requestTimeout: number;
+    symbol: string;
+  }) {
+    return fetch(
+      `https://www.trackinsight.com/search-api/search_v2/${symbol}/_/ticker/default/0/3`,
+      {
+        signal: AbortSignal.timeout(requestTimeout)
+      }
+    )
+      .then((res) => res.json())
+      .then((jsonRes) => {
+        if (
+          jsonRes['results']?.['count'] === 1 ||
+          jsonRes['results']?.['docs']?.[0]?.['ticker'] === symbol
```

Comment:
To support the very basic use case for symbol `VT` with many search results, I have added this additional condition

Reply:
Thanks, make sense! When I was playing with it, perfect match ETFs only returned a single symbol (3+ letter ones) so I thought all would do the same. Thanks again!